Makefile

```diff
@@ -36,7 +36,7 @@ docs-clean:
 	@cd docs; $(MAKE) clean
 
 lint-flake8:
-	flake8 . --ignore D203 --exclude docs/_build
+	flake8 . --ignore D203,E501,F401 --exclude docs/_build
```

Comment:
Since you're messing around with the flake8 linter, please drop the reference to D203. See 76dfc72a0ef0dbfc2b6c55507614b2c3bb3e3c29